### DISCUSSION TITLE
Question about Elastodyn's tower vertical response

### DISCUSSION THREAD

**Discussion by lapadron** [OpenFAST/openfast#981] [Q&A] [answered]
Hello,

Could you please confirm me if Elastodyn doesn't consider vertical deformation when used to model de Tower, i.e., that the tower is assumed rigid in the vertical direction? (I see the horizontal deformations from the TSS and TFA first and second modes, but I don't find nothing related to vertical deformations).

Related to this, could you please give some reference on where I can find the way in which gravitational destiffening is considered in Elastodyn?

Thanks in advance,
Luis

**Comment by jjonkman** [comment, marked as answer]
@lapadron,

ElastoDyn module of OpenFAST considers the nonlinear geometric "shortening" of the tower, i.e., the axial deflection associated with transverse deflection so as to keep the tower length the same when deflected, which is important to gravitational destiffening, but ElastoDyn does not consider elastic compression/stretching of the tower (through an axial E*A stiffness).  The ElastoDyn output that shows the axial "shortening" effect is TTDspAx (compare this to TTDspFA or TTDspSS).

The tower "shortening" effect is documented in one of my first NREL publications: https://www.nrel.gov/docs/fy04osti/35077.pdf 😄.

Best regards,

**Reply by lapadron** [reply]
Thank you!
Best regards,
